samples/nrf_desktop/src/modules/power_manager.c

```diff
@@ -152,17 +158,18 @@ enum power_states sys_pm_policy_next_state(s32_t ticks)
 
 static void system_off(void)
 {
-	/* Port events are needed to leave system off state. */
-	nrf_gpiote_int_enable(GPIOTE_INTENSET_PORT_Msk);
-	irq_enable(GPIOTE_IRQn);
-
-	/* Make sure that port events are enabled before
-	 * the system goes off.
-	 */
-	__DSB();
+	if (power_state != POWER_STATE_ERROR_SUSPENDED) {
+		/* Port events are needed to leave system off state. */
+		nrf_gpiote_int_enable(GPIOTE_INTENSET_PORT_Msk);
```

Comment:
I think this can be thrown away now. Originally the implementation of GPIO driver in zephyr was different. It was reimplemented 9 months ago or so but there was no follow up here.

At the moment this enable is done in `gpio_nrfx_init` and never disabled.

Reply:
removed